### ISSUE TITLE
Experian agreement page needs a `<main>` element

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4074]
## Area or Component
Experian agreement page

## Quick Summary
There's no `<main>` element on the Experian agreement page, so many elements are not covered by landmarks.

## Accessibility Violation
Document should have one main landmark: https://dequeuniversity.com/rules/axe/4.4/landmark-one-main?application=AxeChrome

## Action Requested
Add a `<main>` landmark to the Experian agreement page

## Screenshots (optional)
![Screen Shot 2022-07-26 at 2 55 57 PM](https://user-images.githubusercontent.com/80282552/181120162-81402dbf-cc53-4c84-bdc3-b2bebd060c06.png)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=543263785&range=70:70

**Comment by emmastephenson** [comment]
Fixed by #4276
